src/renderer/dx/DxFontRenderData.h

```diff
@@ -16,6 +16,17 @@
 
 namespace Microsoft::Console::Render
 {
+    enum class AxisTagPresence : BYTE
+    {
+        AxisTagPresenceNone = 0x00,

```

Comment:
nit: since you used `enum class`, you are _required_ to say `AxisTagPresence::`. This makes the name redundant: `AxisTagPresence::AxisTagPresenceX`.

You could legally leave off `AxisTagPresence` from the beginning and it would not pollute the global namespace.